### PR TITLE
Completing code in transactionRequests API

### PR DESCRIPTION
**Pull request by rmothilal**
fixed response codes and objects
added fixes for tests and test helpers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,18 +59,19 @@
...
-        "immutable": "^4.0.0-rc.12",
+        "immutable": "4.0.0-rc.12",
         "jsdoc": "3.6.2",
         "mustache": "3.0.1",
         "uuid4": "1.1.4"
     },
     "devDependencies": {
         "ava": "2.1.0",
-        "eslint": "5.16.0",
+        "eslint": "6.0.1",
```
Comment (lewisdaly) [inline package.json]: I noticed this project uses eslint instead of standard, which all of the other projects use. Is there a reason for that?

Comment (rmothilal) [reply package.json]: Nope. Just used eslint as it was a part of the auto generated code. Can add in standard

Comment (mdebarros) [reply package.json]: Please do :D

File: src/handlers/transactionRequests.js [hunk 2]
```diff
@@ -15,6 +15,6 @@ module.exports = {
    */
   post: function (request, h) {
     transactionRequest.forwardTransactionRequest(request, Enum.endpoints.TRANSACTION_REQUEST_POST)
-    return h.response().code(201)
+    return h.response().code(202)
```
Comment (mdebarros) [inline src/handlers/transactionRequests.js]: I am feeling more so than ever that we should have a standard lib for HTTP based stuff, i.e:
- standard health check server @lewisdaly 
- standard HTTP error code enums

@lewisdaly @rmothilal let me know what you guys think about this idea.

File: src/handlers/health.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ module.exports = {
    */
   get: async (request, h) => {
     try {
-      let {response, statusCode} = await handler.getHealth()
+      const { statusCode } = await handler.getHealth()
```
Comment (lewisdaly) [inline src/handlers/health.js]: You can replace this with the `central-services-health` library along with the shared HealthCheck implementation:

```js
const HealthCheck = require('@mojaloop/central-services-shared').HealthCheck.HealthCheck
const { defaultHealthHandler } = require('@mojaloop/central-services-health')

const healthCheck = new HealthCheck(packageJson, [])

module.exports = {
  get: defaultHealthHandler(healthCheck)
}

```

**Pull request merged**